packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,116 +2,174 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-$thumb-height: var(--iui-size-m); // 16px
-$track-height: var(--iui-size-2xs); // 4px
-$tick-height: var(--iui-size-s); // 12px
+// #region Slider container
+.iui-slider-container {
+  --_iui-slider-container-gap: var(--iui-size-s);
+  --_iui-slider-rail-color: var(--iui-color-border);
+  --_iui-slider-thumb-border-color: var(--iui-color-border-foreground);
+  --_iui-slider-thumb-color: var(--iui-color-background);
+  --_iui-slider-thumb-cursor: grab;
+  --_iui-slider-thumb-size: var(--iui-size-m);
+  --_iui-slider-tick-size: var(--iui-size-s);
+  --_iui-slider-tick-thickness: 1px;
+  --_iui-slider-track-color: var(--iui-color-border-accent);
+  --_iui-slider-track-thickness: calc(var(--_iui-slider-thumb-size) * 0.25);
+
+  display: grid;
+
+  &[data-iui-orientation='horizontal'] {
+    grid-template-columns: auto 1fr auto;
+    grid-template-areas: 'min content max';
+    align-items: center;
 
-.iui-slider-component-container {
-  display: flex;
-  align-items: flex-start;
+    .iui-slider-min,
+    .iui-slider-max {
+      align-items: center;
+    }
 
-  &.iui-disabled {
-    cursor: not-allowed;
+    .iui-slider-min {
+      margin-inline-end: var(--_iui-slider-container-gap);
```

Comment:
is it possible to use grid gap instead of margin here?

Reply:
I originally was using `gap`, but if the min / max grid-areas are left empty or removed, the gap is still present.

With `gap`:
![Screenshot 2023-09-11 at 4 56 39 PM](https://github.com/iTwin/iTwinUI/assets/849817/c032322a-b7a8-45f7-9ee1-50ee999d329a)
![Screenshot 2023-09-11 at 4 58 45 PM](https://github.com/iTwin/iTwinUI/assets/849817/59058ede-0293-4d83-bb98-a9a8366d11e4)


Without `gap`:
![Screenshot 2023-09-11 at 4 57 49 PM](https://github.com/iTwin/iTwinUI/assets/849817/a4133158-f1de-4ea4-a5a8-e4098caed4ee)

Maybe that's ok and not a big deal.

Reply:
it should be possible to use `gap` with flexbox because we won't need to define the structure upfront (the children would define the structure, so the gap will only be present if children are present).

```css
.iui-slider-container {
  display: flex;
  align-items: center;

  .iui-slider-min {
    order: -1;
  }

  .iui-slider-min {
    order: 1;
  }

  .iui-slider {
    flex: 1;
  }
}
```

is this doable without breaking any other visuals that rely on the grid?

Reply:
`flex` certainly does make the code more minimal, but the only area I can't figure out a solution for is the ticks.
<img width="351" alt="Screenshot 2023-09-20 at 5 45 09 PM" src="https://github.com/iTwin/iTwinUI/assets/849817/ee7568c7-4f7b-4c51-a6a2-bb67239b2973">

Reply:
oh yeah, then don't worry about it. i forgot this was one of the main reasons we chose grid (it allows overlapping grid areas)

Reply:
the only way around it i can think is if you move `iui-slider-ticks` inside `iui-slider`.

Reply:
Without testing this, I _predict_ if we did do that then the slider + ticks would vertically center with the min / max instead of only the slider vertically aligning with the min / max.  I mocked-up how I think that might display incorrectly:
<img width="842" alt="Screenshot 2023-09-21 at 2 40 06 PM" src="https://github.com/iTwin/iTwinUI/assets/849817/c0a22e4f-13ff-4367-8efa-9f830f866f1c">